src/firmware/main/app/world/dribbler.c

```diff
@@ -5,15 +5,17 @@
 struct Dribbler
 {
     void (*set_speed)(uint32_t rpm);
+    void (*coast)(void);
```

Comment:
Curious where you found we need the ability to coast the dribbler?

Reply:
fixed

Reply:
So the dribbler doesn't coast?

Reply:
Whoops, commented on the wrong thing. The dribbler currently has a `coast` function, so I just replicated that functionality since it was used in at least one primitive.